src/utils/lazyReact.tsx

```diff
@@ -16,8 +16,12 @@ const NoopComponent = () => null;
  */
 export function LazyComponent<T extends object = any>(factory: () => React.ComponentType<T>, attempts = 5) {
     const get = makeLazy(factory, attempts);
-    return (props: T) => {
+    const component = (props: T) => {
```

Comment:
this gives it a proper displayName for react devtools
```suggestion
    const LazyComponent = (props: T) => {
```